vision/google/cloud/vision/entity.py

```diff
@@ -64,12 +64,32 @@ def from_api_repr(cls, response):
         description = response['description']
         locale = response.get('locale', None)
         locations = [LocationInformation.from_api_repr(location)
-                     for location in response.get('locations', [])]
+                     for location in response.get('locations', ())]
         mid = response.get('mid', None)
         score = response.get('score', None)
 
         return cls(bounds, description, locale, locations, mid, score)
 
+    @classmethod
+    def from_pb(cls, response):
```

Comment:
So this one is actually a `AnnotateImageResponse` object. So I think it's right already?

I'll double check but I think I hunted around the others in this PR to make sure they were accurately named.